src/feature.js

```diff
@@ -31,7 +31,7 @@ var feature = function (arg) {
 
   var m_this = this,
       s_exit = this._exit,
-      m_selectionAPI = arg.selectionAPI === undefined ? false : arg.selectionAPI,
+      m_selectionAPI = arg.selectionAPI === undefined ? false : !!arg.selectionAPI,
```

Comment:
I have heard good and bad about using double negation (https://www.sitepoint.com/javascript-double-negation-trick-trouble/)